pkg/cloudmap/resource_manager.go

```diff
@@ -26,6 +27,9 @@ const (
 	defaultNamespaceCacheTTL               = 2 * time.Minute
 	defaultServiceCacheMaxSize             = 1024
 	defaultServiceCacheTTL                 = 2 * time.Minute
+
+	nodeRegionLabel           = "failure-domain.beta.kubernetes.io/region"
```

Comment:
let's support "topology.kubernetes.io/region" and "topology.kubernetes.io/zone", these two annotation will replace the beta ones in v1.17 https://github.com/kubernetes/kubernetes/pull/81431